lighthouse-core/audits/content-width.js

```diff
@@ -55,8 +55,8 @@ class ContentWidth extends Audit {
       return '';
     }
 
-    return 'The content scroll size is ' + artifact.scrollWidth + 'px, ' +
-        'whereas the viewport size is ' + artifact.viewportWidth + 'px.';
+    return 'The content scroll size is ' + artifact.innerWidth + 'px, ' +
+        'whereas the viewport size is ' + artifact.outerWidth + 'px.';
```

Comment:
Think this sentence needs updating. The first var is the viewport width and the second is
the size of the browser window. Right?  Not sure you want `outerWidth/Height`. That is inclusive of the window chrome. https://developer.mozilla.org/en-US/docs/Web/API/Window/outerWidth

For "content scroll size", don't you want `scrollWidth` of the content?